classes/classes/Creature.as

```diff
@@ -381,7 +381,7 @@ package classes
 		//MALE STUFF
 		//public var cocks:Array;
 		//TODO: Tuck away into Male genital class?
-		public var cocks:Array;
+		public var cocks:/*Cock*/Array;
```

Comment:
I don't remember why, but when I had an idea to use Vector in such situations, but something stopped me. Some array methods are not possible to use with vectors or something.

Anyways, this should work on both IDE and compiler level; even it is not an official standard.